_posts/2016-06-08-component-relative-paths-in-angular-2.md

```diff
@@ -0,0 +1,305 @@
+---
+layout:     post
+title:      "Component-Relative Paths in Angular 2"
+
+date: 2016-06-08
+imageUrl: '/images/banner/opaque-tokens-in-angular-2.jpeg'
+
+summary: "Creating components in Angular 2 has been improved in many ways. But developers should be careful when using external HTML and CSS files. Component-relative paths are not immediatley intuitive. But with a brief review of the relative issues and constraints, developers can more easily create maintainable, reusable, portable components."
+
+categories:
+  - angular
+
+tags:
+  - angular2
+  - components
+  - relative paths
+
+topic: components
+
+author: thomasburleson
+---
+
+Easily creating components is the most-loved feature of Angular 2. By now you should be familiar with using the @Component annotations to create components. And I bet you are also familiar with the required metadata information such as `selector` and `template`.
```

Comment:
@Component -> `@Component`
annotations -> decorators

"And I bet you are" -> how about something like "You are probably also familiar with .... if not, you might want to read our article on [building a zippy component]"